.ci/.docker-images.yml

```diff
@@ -0,0 +1,389 @@
+---
+# Dockerfile build config for .github/workflows/build-docker-images.yml
+# Schema: .ci/.docker-images.schema.json
+templates:
+
+  - &apm-agent-library-docker-images
+    repository: "elastic/apm-pipeline-library"
+    working_directory: ".ci/docker"
+
+  - &opbeans-docker-images
+    tag: "daily"
+
+  - &apm-agent-python-docker-images
+    name: "apm-agent-python"
+    repository: "elastic/apm-agent-python"
+    working_directory: "tests"
+
+  - &apm-agent-nodejs-docker-images
+    name: "apm-agent-nodejs"
+    repository: "elastic/apm-agent-nodejs"
+    working_directory: ".ci/docker/node-container"
+
+  - &apm-agent-ruby-docker-images
+    name: "apm-agent-ruby"
+    repository: "elastic/apm-agent-ruby"
+    working_directory: "spec"
+
+  - &apm-agent-rum-docker-images
+    repository: "elastic/apm-agent-rum-js"
+
+  - &observability-dev-docker-images
+    repository: "elastic/observability-dev"
+
+#  - &beats-docker-images
+#    # Install gvm (https://github.com/moovweb/gvm#installing)
+#    prepare_script: |
+#      bash < <(curl -s -S -L https://raw.githubusercontent.com/moovweb/gvm/master/binscripts/gvm-installer)
+#      . $HOME/.gvm/scripts/gvm
+#      GO_VERSION="go$(cat ${GITHUB_WORKSPACE}/.go-version)"
+#      gvm install go1.4 -B
+#      gvm use go1.4
+#      export GOROOT_BOOTSTRAP=$GOROOT
+#      gvm install "${GO_VERSION}"
+#      gvm use "${GO_VERSION}"
+#      GOBIN="${GOROOT}/bin"
+#      echo "${GOBIN}" >> $GITHUB_PATH
+#      echo "GOBIN=${GOBIN}" >> $GITHUB_ENV
+#      echo "GOROOT=${GOROOT}" >> $GITHUB_ENV
+#      echo "GOPATH=${GOPATH}" >> $GITHUB_ENV
+#    build_script: "make mage && mage compose:buildSupportedVersions"
+#    push_script: "make mage && mage compose:pushSupportedVersions"
```

Comment:
I think is not used, TBC

Reply:
Should we keep it commented for now?

Reply:
https://github.com/elastic/apm-pipeline-library/pull/1986#discussion_r1059343767